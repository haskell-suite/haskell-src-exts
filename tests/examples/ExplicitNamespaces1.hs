{-# LANGUAGE ExplicitNamespaces #-}
import Data.Array.Repa ( type (++) )
import Data.Array.Repa ( type {- comment here -} (++) )
import Data.Array.Repa (    type {- comment2 here -} (++)  {- and here -} )
