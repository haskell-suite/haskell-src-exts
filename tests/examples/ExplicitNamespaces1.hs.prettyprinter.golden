{-# LANGUAGE ExplicitNamespaces #-}
module Main (main) where
import Data.Array.Repa (type (++))
import Data.Array.Repa (type (++))
import Data.Array.Repa (type (++))
